src/main/java/com/example/onlinebookstore/mapper/CartItemMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.example.onlinebookstore.mapper;
+
+import com.example.onlinebookstore.config.MapperConfig;
+import com.example.onlinebookstore.dto.cart.CartItemRequestDto;
+import com.example.onlinebookstore.dto.cart.CartItemResponseDto;
+import com.example.onlinebookstore.model.CartItem;
+import org.mapstruct.AfterMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+
+@Mapper(config = MapperConfig.class)
+public interface CartItemMapper {
+    CartItem toModel(CartItemRequestDto request);
+
+    CartItemResponseDto toDto(CartItem cartItem);
```

Comment:
```suggestion
    @Mapping(target = "bookId", source = "book.id")
    @Mapping(target = "bookTitle", source = "book.title")
    CartItemResponseDto toDto(CartItem cartItem);
```
try if this can replace your helper method